ab_tool/constants.py

```diff
@@ -6,3 +6,5 @@
 STAGE_URL_TAG = 'intervention_point_url_'
 DEPLOY_OPTION_TAG = 'is_canvas_page_for_track_'
 AS_TAB_TAG = 'open_as_tab_for_track_'
+
+MAX_FILE_UPLOAD_SIZE = 10 * 1000 * 1000 # 10 MB
```

Comment:
This seems like something that should be configurable in a settings file, unless there is a hard limit of 10MB for uploads somewhere?  If so, you should document that somewhere here and in the exceptions module that refers to this limit.